src/Bridge/Symfony/Bundle/DependencyInjection/Configuration.php

```diff
@@ -272,8 +272,26 @@ public function getConfigTreeBuilder()
                         ->end()
                     ->end()
                 ->end()
-
-            ->end();
+                ->arrayNode('open_api')
```

Comment:
```suggestion
                ->arrayNode('openapi')
```

Reply:
Also, there should be a child node "info" under the "openapi" node.

Reply:
or call it openapi_info ?
because this config is not make for overide other node

Reply:
I disagree. We want a proper structure for the configuration. Not a bunch of top-level keys scattered without grouping.

Reply:
`enable_swagger` should be deprecated, and changed to this instead:

```yaml
openapi:
    enabled: true
```